src/dvb.c

```diff
@@ -1873,6 +1932,9 @@ int dvb_get_signal(adapter *ad) {
 
         if (ad->new_gs == OLD_SIGNAL)
             get_signal(ad, &status, &ber, &strength, &snr, &dbvalue);
+
+        if (ad->force_tuner_signal != TUNER_FORCE_NO)
```

Comment:
Thanks, the PR looks great.
Do you want adapt_signal to be applied to both cases (get_signal and get_signal_new) ?

Reply:
Hi @catalinii ,

> Do you want adapt_signal to be applied to both cases (get_signal and get_signal_new) ?

I don't think so! The reason is because the `get_signal_new()` function uses the driver information. And we need to trust the driver. The problem is with the `get_signal()` function in which the driver doesn't indicates the type of data. Then in this case the user requires to "tweak" the information. For example, I have one E2 box with four tuners, and each one requires different tweaks... because it's driving different physical tuners with different drivers. Therefore this PR enables the correct configuration. I hope you will merge it soon.

And please note that the duplicated code (i.e. `get_snr_decibels()` and `get_strength_decibels()`) is already used in `get_signal_new()`. So from my point of view the PR is complete.

Regards.

Reply:
The code above allows it to execute even when new_signal is used

Reply:
> The code above allows it to execute even when new_signal is used

You're right! Now it's fixed. Only executed when calling to "old" `get_signal()`.

Ready to merge it?

Reply:
Yes thx